pkgs/applications/editors/codeblocks/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkg-config file zip wrapGAppsHook3 ];
   buildInputs = [ wxGTK32 gtk3 ]
-    ++ lib.optionals contribPlugins [ hunspell gamin boost ];
+    ++ lib.optionals contribPlugins [ hunspell boost ];
```

Comment:
Looks like it needs also [disabling filemanager](https://github.com/Distrotech/codeblocks/blob/58262dd449801fe91c75924ae1500675032b14f1/configure.ac#L189C15-L189C37) since that pulls in gamin:

```patch
--- a/pkgs/applications/editors/codeblocks/default.nix
+++ b/pkgs/applications/editors/codeblocks/default.nix
@@ -119,7 +119,7 @@ stdenv.mkDerivation rec {
   preConfigure = "substituteInPlace ./configure --replace /usr/bin/file ${file}/bin/file";
   postConfigure = lib.optionalString stdenv.isLinux "substituteInPlace libtool --replace ldconfig ${stdenv.cc.libc.bin}/bin/ldconfig";
   configureFlags = [ "--enable-pch=no" ] ++ lib.optionals contribPlugins [
-    ("--with-contrib-plugins" + lib.optionalString stdenv.isDarwin "=all,-FileManager,-NassiShneiderman")
+    ("--with-contrib-plugins=all,-FileManager" + lib.optionalString stdenv.isDarwin ",-NassiShneiderman")
     "--with-boost-libdir=${boost}/lib"
   ];
   postInstall = lib.optionalString stdenv.isDarwin ''
```